app/controllers/icons_controller.rb

```diff
@@ -55,6 +55,7 @@ def show
       @times_used = (posts_using.count + replies_using.count)
       @posts_used = (posts_using.pluck(:id) + replies_using.select(:post_id).distinct.pluck(:post_id)).uniq.count
     end
+    @galleries = @icon.galleries.ordered_by_name
```

Comment:
Oh that's clever. Although wouldn't it use the association cache anyway?

Reply:
I'm not sure which cache you're gesturing at? But @icon.galleries and @icon.galleries.ordered_by_name wouldn't use the same cache afaict.

Reply:
Huh. I guess that makes sense, and so it makes sense to add this. :+1:

Reply:
Yeah none of the other cases using galleries seemed to care what the order was, so no reason not to be consistent :shrug: